pkg/rbd/nodeserver.go

```diff
@@ -441,6 +481,20 @@ func (ns *NodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 	return &csi.NodeUnpublishVolumeResponse{}, nil
 }
 
+// getStagingTargetPath concats NodeUnstageVolumeRequest's target path with
+// with the volumeID
+func getStagingTargetPath(req interface{}) string {
```

Comment:
need to update function comment, its says NodeUnstageRequest, this function is doing for both stage and unstage